dspace-api/src/main/java/org/dspace/discovery/IndexClient.java

```diff
@@ -111,6 +127,8 @@ public static void main(String[] args) throws SQLException, IOException, SearchS
         options.addOption(OptionBuilder.isRequired(false).withDescription(
             "optimize search core").create("o"));
 
+        options.addOption("e", "readfile", true, "Read the identifier from a file");
```

Comment:
These identifiers need to be items, best to alter the documentation here to make this clear.

Reply:
option removed as it is out of scope of this PR